src/record/json-path.js

```diff
@@ -124,6 +124,6 @@ function tokenize( path ) {
 	}
 
 	return cache[ path ] = parts.map( function( part ) {
-		return !isNaN( part ) ? parseInt( part, 10 ) : part;
+		return part;
```

Comment:
Hm actually, parts is created just a few lines above it, and isn't sent anywhere. So I guess I don't even need to copy it. I'll just `cache[ path ] = parts`